yt_dlp/extractor/hytale.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from .common import InfoExtractor
+
+
+class HytaleIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?hytale\.com/news/20\d\d/(0?[1-9]|1[012])/(?P<id>[a-z0-9-]+)'
+    _TESTS = [{
+        'url': 'https://hytale.com/news/2021/07/summer-2021-development-update',
+        'info_dict': {
+            'id': 'summer-2021-development-update',
+            'title': 'Summer 2021 Development Update',
+        },
+        'playlist_count': 4,
+    }]
+
+    _MD5_REGEX = r'<stream class="ql-video cf-stream" src="([a-z0-9]{32})"'
```

Comment:
this is not safe

do:
```suggestion
    _MD5_REGEX = r'<stream\s*class\s*=\s*"ql-video\s*cf-stream"\s*src\s*=\s*"([a-f0-9]{32})"'
```

MD5 uses hex so a-f is enough